### PR TITLE
Cura 6853 layer slider behind action panel

### PR DESCRIPTION
**Pull request by nallath**

### CODE REVIEW

File: plugins/SimulationView/LayerSlider.qml [hunk 1]
```diff
@@ -333,7 +338,7 @@ Item
         // set the slider position based on the lower value
         function setValue(value)
         {
-
+            print("lower handle set value: " + value)
```
Comment (nallath) [inline plugins/SimulationView/LayerSlider.qml]:
```suggestion
```

File: plugins/SimulationView/SimulationViewMainComponent.qml [hunk 2]
```diff
@@ -209,6 +216,7 @@ Item
         // Make sure the slider handlers show the correct value after switching views
         Component.onCompleted:
         {
+            print("paneltop", panelTop, "screenscaleFactor")
```
Comment (nallath) [inline plugins/SimulationView/SimulationViewMainComponent.qml]:
```suggestion
```

**Comment by nallath** [comment]
I'm not really a fan of this solution, but apart from changing the actual location of those sliders, I don't really see any other way to fix this. 

Changing the order of the loader and the actionPanelWidget in previewMain at least ensures that both of them are usable, but then it simply lets the sliders overlap the actionPanelWidget.

**Comment by ninovanhooff** [comment]
I removed the dependency on external code. Kindly reconsider.

**Pull request merged**